controllers/foodsController.js

```diff
@@ -71,6 +74,13 @@ const destroy = async (req, res) => {
   }
 }
 
+const sanitizeEntry = (userEntry) => {
```

Comment:
Remember the time we said we'd move this to another file? Maybe we still will someday.